src/synchronization/panZoomSynchronizer.js

```diff
@@ -1,6 +1,13 @@
 import external from '../externalModules.js';
 
-// This function synchronizes the target zoom and pan to match the source
+/**
+ * Synchronize the target zoom and pan to match the source
+ *
+ * @param {Synchronizer} synchronizer - The Synchronizer instance that attaches this
```

Comment:
If you want to use `{Synchronizer}` throughout instead of `{Object}`, we will also require a [`@typedef`](http://usejsdoc.org/tags-typedef.html) for `Synchronizer`.

Feel free to just use `{Object}` for now, so we can merge this and the definition can made further down the line (in a different PR).